commands/provider/assets/readme_template.md

```diff
@@ -23,9 +23,24 @@ You can install MESG Core by running the following command or [follow the instal
 bash <(curl -fsSL https://mesg.com/install)
 ```
 
-## Service
+{{if .Sid}}
+## Purchase the service
 
-{{if .Repository}}To deploy this service, run the following command:
+To deploy this service, you first need to purchase one of its offers from the MESG Marketplace.
+
+To do so, go to the service's page on the MESG Marketplace and check the offers tab: https://marketplace.mesg.com/services/{{.Sid}}#offers
+{{end}}
+
+## Deploy the service
```

Comment:
I agree with https://github.com/mesg-foundation/core/pull/954/files#r283428257, can you remove the `## Deploy the service` block to only keep `## Purchase the service`.

I think it's best to simply remove the deploy for git url to only show the marketplace deploy

Reply:
Just like: https://github.com/mesg-foundation/service-ethereum-erc20#purchase-the-service